cis194/week7/haotian/week7/src/Scrabble.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module Scrabble where
+
+import Data.Monoid
+import Data.Char
+
+newtype Score = Score Int
+  deriving (Eq, Read, Show, Ord, Num)
+
+instance Monoid Score where
+  mempty  = Score 0
+  mappend = (+)
```

Comment:
This is duplicated definition.

Reply:
explain to me why? I am dumb. 😄 

does it because Monoid belongs to Semigroup?

Reply:
`mappend` is `(<>)` which is already defined below

Reply:
@stevemao great! thanks